### PR TITLE
Prevent blink counter from overwriting itself. Document blinks...

### PR DESCRIPTION
**Pull request by CameronRP**
- Prevent blink counter from overwriting itself.
- Document blinks meanings in code.
- Reformat i2c receive function.

### CODE REVIEW

File: attiny/attiny.ino [hunk 1]
```diff
@@ -125,35 +133,40 @@ void requestEvent() {
...
+    setBlinks(10);
     return;
   }
-  byte message[10];
-  int i = 0;
-  while(TinyWireS.available()) {
-    message[i++] = TinyWireS.receive();
-  }
-  switch(message[0]) {
+  bool successfullMessageRead = false;
```
Comment (mjs) [inline attiny/attiny.ino]: "successful" has one "l". Also, the name is a bit long. What about `successfulRead` or `messageOk`?

File: attiny/attiny.ino [hunk 2]
```diff
@@ -170,6 +183,12 @@ void testBlink(int x) {
   }
 }
 
+void setBlinks(int i) {
+  if (blinks <= 0) {
+    blinks = i;
+  }
```
Comment (mjs) [inline attiny/attiny.ino:189]: I guess it's ok that if 2 blink events happen close together the second one is lost?

File: attiny/attiny.ino [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
 // The two main functions of this is to power off the raspberry pi during the day. This is done by the raspberry pi telling the attiny through i2c how long to be turned off for.
 // The attiny then turns the pi off using a mosfet. And will power it back on when needed.
 // The second function is to work as a WDT for the raspberry Pi when is is runnign through the nihgt. So if the pi doesn't reset the WDT the attiny will power off and on the pi.
 
+// There is a power LED that will pulse when the code is running. This LED can also blink to help debugging. Below are a list of what the different number of blinks mean.
+// 1 Blink: Reset Pi WDT.
+// 2 Blinks: Finished setup.
+// 3 Blinks: Pi WDT failed.
+// 4 Blinks: Starting Pi sleep.
+// 10 Blinks: Error with reading i2c message.
```
Comment (mjs) [inline attiny/attiny.ino:12]: Thanks for doing this.

**Review by mjs — APPROVED** [review]
Nice. Just little stuff.

File: attiny/attiny.ino (hunk 2 above)
Comment (CameronRP) [reply attiny/attiny.ino:189]: Yeah, we could buffer the blinks.. But this at least prevents getting wrong messages.

**Pull request merged**